### ISSUE TITLE
accessibility issues

### ISSUE THREAD

**Issue by RichMorin** [beam-community/elixir-companies#522]
The site makes extensive use of icons to indicate the nature of accompanying links and text, etc. There should be tooltips to make the meaning accessible via screen readers, etc. More generally, running something like Chrome's Accessibility Audit might be useful.

**Comment by doomspork** [comment]
@burden / @tajchumber either of you interested in grabbing this?
